lib/Facebook.js

```diff
@@ -24,7 +24,8 @@ function Facebookbot(configuration) {
                     if (convo) {
                         for (var s = 0; s < convo.sent.length; s++) {
                             if (convo.sent[s].sent_timestamp <= message.delivery.watermark ||
-                                (convo.sent[s].api_response && convo.sent[s].api_response.mid == mid)) {
+                              (convo.sent[s].api_response &&
+                                  (convo.sent[s].api_response.mid || convo.sent[s].api_response.message_id) == mid)) {
```

Comment:
I don't see where ````api_response.mid```` is used ! maybe it should be removed !

Reply:
@ouadie-lahdioui I don't see where `api_response.mid` is used either, but it was there.

I'm unclear if this ever worked?!? I can remove the check for `api_response.mid`, but I don't want to do it unless directed, since there might have been a reason for it in the past.